tests/baselines/reference/asyncAwaitNestedClasses_es5.js

```diff
@@ -34,11 +34,11 @@ var A = /** @class */ (function () {
             }
             C.func = function () {
                 return __awaiter(this, void 0, void 0, function () {
-                    return __generator(this, function (_b) {

```

Comment:
I'm not clear why this changed, aside from the possibility of the missing call to `endBlockScope` I mentioned elsewhere.